src/de/grobox/liberario/fragments/DirectionsFragment.java

```diff
@@ -533,7 +564,7 @@ private void showMore(boolean animate) {
 
 	private void showLess(boolean animate) {
 		showingMore = false;
-		ui.via.setLocation(null);
+		ui.via.setLocation((Location)null);
```

Comment:
When I was putting this together it was. I will check if or how this can be removed now.